### ISSUE TITLE
Color Emoji produces ghosting effect when scrolling through the window

### ISSUE THREAD

**Issue by chansey97** [mintty/mintty#1261]
Following the [wiki tips](https://github.com/mintty/mintty/wiki/Tips#emojis) to config Color Emoji.

It can display emojis, but produces ghosting effect when scrolling through the window.

Please refer to the screen recording below.

https://github.com/mintty/mintty/assets/4015056/a30c6dbf-eb1a-4f34-a59c-40ee96e40939

P.s. After select all content and cancel, the rendering will return to normal. It does not seem that the screen refresh properly.

Version: mintty 3.7.0  (x86_64-pc-msys) 

Thanks.

**Comment by mintty** [comment]
Please upload your test file. Also which font, font size, and emoji style do you run?

**Comment by chansey97** [comment]
> Please upload your test file.

[unicode.txt](https://github.com/mintty/mintty/files/14970457/unicode.txt)


> Also which font, font size, and emoji style do you run?

.minttyrc

```
Term=xterm
StatusLine=yes
Locale=en_US
Charset=UTF-8

ShowHiddenFonts=yes

Font=Source Code Pro
FontWeight=400
FontHeight=12

Charwidth=unicode
Emojis=zoom
EmojiPlacement=full
```

It can be reproduced on both windows 7 and 10. 

P.s. Why I mentioned about Windows 7, even though it is no longer officially supported? Because AFAIK mintty is the only Terminal emulator that supports Color Emoji on Windows 7. So I hope to fix it.

**Comment by mintty** [comment]
Tweaked handling of setting EmojiPlacement=full.
Also enabled emoji space expansion for numbers and flag letters, even without that setting.

**Comment by mintty** [comment]
Released 3.7.2.
